packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -22,6 +21,7 @@ module.exports = {
 
     const permissionQuery = await permissionChecker.sanitizedQuery.read(query);
 
+    // @ts-ignore TODO
```

Comment:
In this case probably it's not even important, but I would normally use ts-expect-error instead of ts-ignore. While it can be more annoying somethimes , if we leave the message there and the original ts errors disappears, it will warn us